core/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -944,6 +944,14 @@ public XContentBuilder percentageField(String rawFieldName, String readableField
         return this;
     }
 
+    public XContentBuilder nanosAsMillis(String rawFieldName, String readableFieldName, long nanos) throws IOException {
```

Comment:
Good point, the "machine readable" value is printed out "as is" (that its a nanos time value is actually only clear by the field and variable names). The human readable string assumes the input is nanos and prints its as millis (adding "ms" suffix, dividing by the appropriate constant). Thats why I choose that name, but now I start to think that this formatting is probably only useful for the two cases in the Profile output and we should makes this more generic? 
I was thinking about putting it somewhere else (some static util in one of the Profile response classes), but since this method deals with the "humanReadable" flag its probably best here alongside the similar other methods? 
One idea would make the signature (String rawFieldName, String readableFieldName, long rawLong, String formattedLong) and keeping the formatting in the respective classes, wdyt? Then the name could be sth. like "formattedLongField"?